Sources/Common/Core/DataArray/index.d.ts

```diff
@@ -0,0 +1,101 @@
+interface VtkStatisticInformation {
+  min: number;
+  max: number;
+  count: number;
+  sum: number;
+  mean: number;
+}
+
+interface VtkRange {
+  min: number;
+  max: number;
+}
+
+interface VtkRangeHelper {
+  add: (value: number) => void;
+  get: () => VtkStatisticInformation;
+  getRange(): () => VtkRange;
+}
+
+// ----------------------------------------------------------------------------
+// Static API
+// ----------------------------------------------------------------------------
+
+export const STATIC = {
+  computeRange: (values: Array<number>, component: number = 0, numberOfComponents: number = 1) => VtkRange,
```

Comment:
Default values should not appear in function signatures. Instead of it should be replace by optionnals
For examples :
```suggestion
  computeRange: (values: Array<number>, component?: number, numberOfComponents?: number) => VtkRange,  
```
As you can see in the screenshot, visual code replace it automatically when displaying signature
![image](https://user-images.githubusercontent.com/18531147/81474084-76911c00-9203-11ea-8f81-32e05dd0fd41.png)


![image](https://user-images.githubusercontent.com/18531147/81474134-b22be600-9203-11ea-9024-639a1a6013a8.png)

Reply:
Fun fact, to reveal all the problems in visual code I need to close all js files